packages/dashboard/src/components/Card/BodyText/BodyText.styles.ts

```diff
@@ -3,14 +3,14 @@ import { IBodyTextStyles } from './BodyText.types';
 export const getStyles = (): IBodyTextStyles => {
   return {
     root: {
-      fontFamily: 'Segoe UI',
+      fontFamily: '"Segoe UI Regular WestEuropean","Segoe UI","Segoe WP",Tahoma,Arial,sans-serif',
```

Comment:
instead of hardcoding this, you could just pull FontStyles from the theme.

theme.fontStyles.medium